netbout-web/src/test/java/com/netbout/rest/account/TkIndexTest.java

```diff
@@ -55,13 +56,17 @@ public void rendersPage() throws Exception {
         base.user(URN.create(urn)).aliases().add(alias);
         MatcherAssert.assertThat(
             new RsPrint(
-                new TkIndex(base).act(new RqWithAuth(urn))
-            ).printBody(),
+                new TkIndex(base).act(
+                    new RqWithHeader(
+                        new RqWithAuth(urn),
+                        "Accept",
+                        "text/xml"
+                    )
+                )).printBody(),
```

Comment:
@exper0 let's move the final `).printBody(),` to the next line to maintain correct indentation.